CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ CHANGELOG
 
 - Add the .NET Core 3.0 runtime to the pulumi/pulumi container. [#3616](https://github.com/pulumi/pulumi/pull/3616).
 
+- `Output.apply` (for the JS, Python and .Net sdks) has updated semantics, and will lift dependencies from inner Outputs to the returned Output.
+  https://github.com/pulumi/pulumi/pull/3663
```

Comment:
```suggestion
  [#3663](https://github.com/pulumi/pulumi/pull/3663)
```

Reply:
Let’s make this consistent with all other entries and keep on the same line.